### PR TITLE
Artifact chem trigger expansion

### PR DESCRIPTION
**Pull request by amylizzle**
## About the PR 
Adds carbon and silicon touch activating chems.

## Why's this needed? 
I want to activate all the artifacts with a smoke bomb.

## Changelog

```changelog
(u)Amylizzle
(+)Carbon and silicon touch artifacts can be activated with appropriate life-related and robot-related chems now.
```

### CODE REVIEW

**Comment by Chatauscours** [comment]
I would revert the [cc46fb1] commit, as it seems this commit are the changes introduced by your previous PR about syndie frames (#8409).

**Comment by amylizzle** [comment]
ah shit, one sec

File: code/obj/artifacts/artifactprocs.dm [hunk 1]
```diff
@@ -184,7 +184,14 @@
 		return
 	var/datum/artifact/A = src.artifact
 	switch(reagent_id)
-		if("radium","porktonium")
+		if("porktonium")
+			src.ArtifactStimulus("radiate", round(volume / 10))
+			src.ArtifactStimulus("carbtouch", round(volume / 5))
+		if("carbon","synthflesh","blood","bloodc","meat_slurry")
+			src.ArtifactStimulus("carbtouch", round(volume / 5)) //require at least 5 units
+		if("silicon","silicon_dioxide","nanites","corruptnanites","silicate")
```
Comment (zjdtmkhzt) [inline code/obj/artifacts/artifactprocs.dm:192]:
```suggestion
		if("silicon","silicon_dioxide","nanites","corruptnanites","goodnanites","silicate")
```
I think this is the id for directed nanites, seems like it'd fit in?

**Comment by Azrun** [comment]
This is neat.

Subjective personal opinion: I think making this work for carbon and silicon (maybe even silicon dioxide) feels like it would devalue those triggers. Blood and nanites feels good as those are representative of life and higher function.

My head cannon is they are waiting to be interacted with be some being... not tumbling across a quarry.

**Comment by amylizzle** [comment]
> This is neat.
> 
> Subjective personal opinion: I think making this work for carbon and silicon (maybe even silicon dioxide) feels like it would devalue those triggers. Blood and nanites feels good as those are representative of life and higher function.
> 
> My head cannon is they are waiting to be interacted with be some being... not tumbling across a quarry.

Agreed and done. I feel like silicon touch could use some easier to access chems though.

**Pull request merged**